### ISSUE TITLE
Events page responsiblity

### ISSUE THREAD

**Issue by Compro-Prasad** [lugnitdgp/lugnitdgp.github.io#31]
The events page i.e. http://lugnitdgp.github.io/events.html is not having correct information on the events. And moreover someone has to come forward to update the events page as soon as its decided. Since, it needs admin rights to add an event kindly request for it if you are interested.

**Comment by divyanshumehta** [comment]
@Compro-Prasad what admin rights are needed ? . Like the event co-ordinator can update the information regarding the events. Also we can provide for feedback links under all events.

**Comment by Compro-Prasad** [comment]
Yes, the events will be edited from netlify's admin portal presently whose access is limited to some members in our club.

**Comment by ghoshbishakh** [comment]
@vartika-arora can you go through the events and clean them. Posters need to be updated and also the descriptions.

**Comment by vartika-arora** [comment]
@ghoshbishakh Okay
